.github/CONTRIBUTING.md

```diff
@@ -64,7 +64,7 @@ $ ( cd integration && ../script/glide.sh get github.com/baz/quuz )
 # generate (Only required to integrate other components such as web dashboard)
 $ go generate
 # Standard go build
-$ go build
+$ go build ./cmd/traefik
 # Using gox to build multiple platform
 $ gox "linux darwin" "386 amd64 arm" \
```

Comment:
I'm wondering if we should kick this one out. Personally, I've never used it, but maybe it's just me...